PostHog/Replay/PostHogSessionReplayConfig.swift

```diff
@@ -56,6 +56,43 @@
         /// Note: Previously `debouncerDelay`
         @objc public var throttleDelay: TimeInterval = 1
 
-        // TODO: sessionRecording config such as networkPayloadCapture, captureConsoleLogs, sampleRate, etc
+        /// Enable capturing console output for session replay.
+        ///
+        /// When enabled, logs from the following sources will be captured:
+        /// - Standard output (stdout)
+        /// - Standard error (stderr)
+        /// - OSLog messages
+        /// - NSLog messages
+        ///
+        /// Each log entry will be tagged with a level (info/warning/error) based on the message content
+        /// and the source. The level detection uses `captureLogsErrorPattern` and `captureLogsWarningPattern`.
+        ///
+        /// Defaults to `false`
+        @objc public var captureLogs: Bool = false
+
+        /// Regular expression pattern used to identify error-level log messages.
+        ///
+        /// The pattern is applied to each log message to determine if it should be tagged as an error.
+        /// By default, it matches common error indicators such as:
+        /// - The word "error", "exception", "fail" or "failed"
+        /// - OSLog messages with type "Error" or "Fault"
+        ///
+        /// You can customize this pattern to match your application's logging format.
+        ///
+        /// Messages that don't match either the `error` or `warning` patterns are tagged as `info` level.
+        @objc public var logMessageErrorPattern = "(error|exception|fail(ed)?|OSLOG-.*type:\"Error\"|OSLOG-.*type:\"Fault\")"
```

Comment:
whats about having just one? `logMessagePattern` that includes both things
or even logging everything non-system-logging? the FE has filters in place already